ci-operator/step-registry/ipi/conf/azurestack/creds/ipi-conf-azurestack-creds-commands.sh

```diff
@@ -73,8 +59,8 @@ do
       continue
   fi
 
-  filename=manifest_${SECRET_NAMESPACE}_${SECRET_NAME}_secret.yml
-  cat >> "${SHARED_DIR}/${filename}" << EOF
+  filename=manifest_${SECRET_NAMESPACE}_secret.yml
+  cat >> "${SHARED_DIR}/requests/${filename}" << EOF
```

Comment:
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/34879/rehearse-34879-pull-ci-openshift-installer-master-e2e-azurestack/1618966959116783616
` /bin/bash: line 64: /tmp/secret/requests/manifest_openshift-cloud-controller-manager_secret.yml: No such file or directory `

Reply:
Has this been resolved?